.travis.yml

```diff
@@ -10,6 +10,10 @@ matrix:
           env: TOX_ENV=py34
         - python: 3.5
           env: TOX_ENV=py35
+        - python: pypy3
+          env: TOX_ENV=pypy3
+before_install:
+    - "if [[ $TRAVIS_PYTHON_VERSION == 'pypy3' ]]; then deactivate && cd $HOME && wget https://bitbucket.org/squeaky/portable-pypy/downloads/pypy3.5-5.7-beta-linux_x86_64-portable.tar.bz2 && tar -jxvf pypy3.5-5.7-beta-linux_x86_64-portable.tar.bz2 && cd pypy3.5-5.7-beta-linux_x86_64-portable && echo 'Setting up aliases...' && export PATH=$HOME/pypy3.5-5.7-beta-linux_x86_64-portable/bin/:$PATH && ln -s ~/pypy3.5-5.7-beta-linux_x86_64-portable/bin/pypy3.5 bin/python && echo 'Setting up pip...' && bin/pypy -m ensurepip && ln -s ~/pypy3.5-5.7-beta-linux_x86_64-portable/bin/pip3.5 bin/pip && cd $TRAVIS_BUILD_DIR; fi"
```

Comment:
can we turn this into a shell script in some .travis directory perhaps? then we'd just invoke it here.